YACS-RCOS/yacs.n
# Add school property to courses

**Issue**

Closes #162 
Closes #163 

**Database Changes/Migrations**

Added column `school` to `courses` table to indicate the school of the course

**Test Modifications**

Added test `tests/rpi_data/test_add_school_column.py` that tests that all schools collected by `SchoolDepartmentMapper` are unique and a few classes for basic functionality

**Test Procedure**

1. Update db schema
2. Load courses into db using csvs
3. Start api
4. Make request to `/api/class` which should show courses with new additional property `school`

**Photos**

N/A

**Additional Info**
